src/core/map.cpp

```diff
@@ -682,19 +682,37 @@ bool Map::exportTo(const QString& path, MapView* view, const FileFormat* format)
 	bool success = false;
 	if (file.open(QIODevice::WriteOnly))
 	{
-		try
+		if (exporter->noIoStream())
```

Comment:
I don't fee comfortable with the two similar blocks (in `if`and `else`) following this line. However I can't recommend a particular change on the moment. I would like to come to this later.

(For GDAL 2.0, we might keep using QSaveFile if we can leverage [`/vsistdout/`](http://gdal.org/gdal_virtual_file_systems.html#gdal_virtual_file_systems_vsistdout) with `VSIStdoutSetRedirection()`. Alternatively, we might explicitly support file formats which operate directly on paths. Maybe we need the latter anyway to leverage the full potential of GDAL for import and export. As a third, long-term option, one could contribute to GDAL another `/vsi.../` which uses Qt.)